src/opening-period/page/CreateNewOpeningPeriodPage.scss

```diff
@@ -0,0 +1,122 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.add-new-opening-period-page-header-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.add-new-opening-period-page-title {
+  margin-top: $spacing-2-xs;
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+  font-size: $fontsize-heading-m;
+  color: var(--color-coat-of-arms-blue);
+}
+
+.add-new-opening-period-form {
+  label {
+    display: block;
+    margin-top: $spacing-m;
+  }
+
+  input {
+    display: block;
+  }
+
+  h3 {
+    margin-bottom: 0;
+  }
+
+  margin-top: $spacing-xl;
+}
+
+.opening-period-optional-description {
+  padding: $spacing-layout-2-xs;
+  width: calc(0.5808333 * 100%);
+  height: 164px;
+  border: 2px solid rgb(128, 128, 128);
+  margin-bottom: $spacing-layout-2-xs;
+
+  @media screen and (max-width: $breakpoint-m) {
+    width: 100%;
+  }
+}
+
+.opening-period-section-title {
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
```

Comment:
Would it be easier to add into index.scss:
```css
h1, h2, h3, h4, h5, button, label, input {
   font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
}

```

Reply:
Ah good point, specially if those all actually are with the same font-family! I'll check and will make the refactoring if that is the case. This was a good call!

- [ ] Check if common font-family is possible for h1, h2, h3, h4, h5, button, label, input elements

Reply:
It might be actually:
```css
h1, h2, h3, h4 {
 font-family:  bold..
}

button, label {
 font-family: default..
}
```

Reply:
Checked and it seems we could change the h-elements under common font-family. About the other elements, I fear there is a risk that it might affect also to all third party components (hds for example), and it could lead to some surprising side-effects?